lib/index.html

```diff
@@ -138,14 +137,14 @@ <h4>Interactive</h4>
 			</div>
 
 			<div class="features-additional">
-				<img src="images/icon-customcode.png" aria-hidden="true">
+				<img src="images/icon-customcode.png" aria-hidden="true" alt="" role="presentation">
```

Comment:
I don't have a good alt text for this one and its already `aria-hidden` but we also should not have empty alt texts. If we don't have a text for them and they are hidden then there should probably not be a alt text at all

Reply:
A11y says something different here: 
"Use an empty alt attribute for any image that is decorative or not necessary for understanding the content of the page"
https://a11yproject.com/posts/alt-text/

Reply:
Ok i just read about it again and webaim.org for example says that we should leave it empty.
Strange but we can do it this way